.github/workflows/presubmit.yml

```diff
@@ -13,17 +13,18 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Setup Ninja
+      - name: Setup Dependencies
         shell: bash
         run: |
           set -x
-          sudo apt install ninja-build
-      - name: Setup environement variables
+          sudo apt install meson ninja-build
```

Comment:
I thought meson was already there, is this needed to get the latest version or something?

Reply:
In fact, meson should be installed from checkout.sh, so let's remove it from here.
Ninja is needed for every project and is installed here, while the project dependencies are managed from the checkout scripts.

Reply:
If you add a single commit on top of your branch, the CI should use the cache.